packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -184,4 +191,13 @@ export class MonacoEditorProvider {
         };
     }
 
+    createDiffNavigator(editor: TextEditor): DiffNavigator {
```

Comment:
protected?

Reply:
it's called from factory, cf. https://github.com/theia-ide/theia/blob/d824d61e809fe41c2fffd3ff7f08779ba95800c7/packages/monaco/src/browser/monaco-frontend-module.ts#L55